docs/event-bus/docs/013-service-programming-model.md

```diff
@@ -0,0 +1,76 @@
+
+![TEST](assets/service-programming-model.png)
+
+The event is delivered as an `HTTP POST` request. Event Metadata is a part of HTTP request Headers. Event Payload is the body of the request.
+
+## Event Metadata
+
+Following HTTP headers provide information about the event metadata.
+
+|Header| details|
+|------|--------|
+| kyma-event-id | This is the business event's ID which has been delivered to the micro-service. e.g. `8365633f-53d1-4518-85e7-e5604839bca7` |
+| kyma-event-time | This is the business event's time which has been delivered to the micro-service. e.g. `2018-11-02T22:08:41+00:00` |
+| kyma-event-type | This is the business event's type which has been delivered to the micro-service. e.g. `order.created` |
+| kyma-event-type-version | This is the business event's version which has been delivered to the micro-service. e.g. `v1` |
+| kyma-source-environment | This is the business event's source environment which has been delivered to the micro-service. e.g. `ec-qa` |
+| kyma-source-namespace | This is the business event's source namespace which has been delivered to the micro-service. e.g. `local.kyma.commerce` |
+| kyma-source-type | This is the business event's source type which has been delivered to the micro-service. e.g. `commerce` |
+| kyma-subscription | This is the subscription name which is defined in the subscription contract/CRD and this business event has been published to its subscribers. e.g. `example-subscription` |
+| kyma-topic | This is the topic name used internally by the kyma event bus to deliver the messages to the subscribers via a message bus. e.g. `ec-qa.local\.kyma\.commerce.commerce.test-event-bus.v1` |
+| x-b3-flags | This header is used by the Zipkin tracer in Envoy. The encode one or more options. For example, Debug is encoded as X-B3-Flags: 1. See more on zipkin tracing [here](https://github.com/openzipkin/b3-propagation). e.g. `0` |
+| x-b3-parentspanid | This header is used by the Zipkin tracer in Envoy. The ParentSpanId is 64-bit in length and indicates the position of the parent operation in the trace tree. When the span is the root of the trace tree, the ParentSpanId is absent. See more on zipkin tracing [here](https://github.com/openzipkin/b3-propagation) e.g. `3f52e7d591bf74e8` |
+| x-b3-sampled | This header is used by the Zipkin tracer in Envoy. When the Sampled flag is either not specified or set to 1, the span will be reported to the tracing system. Once Sampled is set to 0 or 1, the same value should be consistently sent downstream. See more on zipkin tracing [here](https://github.com/openzipkin/b3-propagation). e.g. `1` |
+| x-b3-spanid | This header is used by the Zipkin tracer in Envoy. The SpanId is 64-bit in length and indicates the position of the current operation in the trace tree. The value should not be interpreted: it may or may not be derived from the value of the TraceId. See more on zipkin tracing [here](https://github.com/openzipkin/b3-propagation). e.g. `7350c92dc6fa825e` |
```

Comment:
``Header used by the Zipkin tracer in Envoy. The **SpanId** is 64-bit in length and indicates the position of the current operation in the trace tree. The value should not be interpreted. It may or may not be derived from the value of the **TraceId**.``